quinn-udp/benches/throughput.rs

```diff
@@ -39,7 +43,7 @@ pub fn criterion_benchmark(c: &mut Criterion) {
 
         let transmit = Transmit {
             destination: dst_addr,
-            ecn: None,
+            ecn: Some(quinn_udp::EcnCodepoint::Ect0),
```

Comment:
Is this relevant to throughput testing? We already cover the ability to send/receive ECN in the unit tests.

Reply:
Sorry this snuck in from local testing.